tests/profiles.py

```diff
@@ -32,6 +33,8 @@ def _build_databricks_cluster_target(
     }
     if catalog is not None:
         profile["catalog"] = catalog
+    if schema is not None:
+        profile["schema"] = schema
```

Comment:
Even if the schema is set here doesn't it just get overwritten with the result of the unique_schema fixture in the dbt_profile_data fixture? That's what happens with I try this.

Reply:
Thank you! I overwrite the unique_schema and some test is passed . But several test is faield

https://github.com/databricks/dbt-databricks/pull/538/commits/c4b9f8bcfc76b580798fcfcc7e20fd08d63c137c

```
.tox/integration-databricks-uc-sql-endpoint/bin/python -m pytest -v -m profile_databricks_uc_sql_endpoint -n4 tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint
====================================================== test session starts =======================================================
platform darwin -- Python 3.11.5, pytest-7.4.3, pluggy-1.3.0 -- /Users/local_path/Desktop/workspace/dbt-databricks/.tox/integration-databricks-uc-sql-endpoint/bin/python
cachedir: .pytest_cache
rootdir: /Users/local_path/Desktop/workspace/dbt-databricks
configfile: pytest.ini
plugins: csv-3.0.0, flaky-3.7.0, dotenv-0.5.2, xdist-3.5.0
4 workers [1 item]      
scheduling tests via LoadScheduling

tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint 
[gw1] [100%] PASSED tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint 

======================================================== warnings summary ========================================================
tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint
tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint
tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint
tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint
tests/integration/incremental_strategies/test_incremental_strategies.py::TestBadStrategies::test_bad_strategies_databricks_uc_sql_endpoint
  /Users/local_path/Desktop/workspace/dbt-databricks/dbt/adapters/databricks/connections.py:693: UserWarning: The cursor was closed by destructor.

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
================================================= 1 passed, 5 warnings in 8.62s ==================================================

```